### PR TITLE
feat: telemetry consent modal

### PR DESCRIPTION
**Pull request by 404-Program-not-found**
Added a cookie consent popup on the first visit
screenshot

### CODE REVIEW

**Comment by oSumAtrIX** [comment]
The button actions should switch places. Left = yes, right = no

**Comment by 404-Program-not-found** [comment]
updated

**Comment by kazimmt** [comment]
I think you should align the buttons on the Right side
Like this:
IMG_20230607_093049

**Comment by 404-Program-not-found** [comment]
> I think you should align the buttons on the Right side
> Like this:
> IMG_20230607_093049
> 

The position is purposefully not aligned the same so users don't spam through accept

**Comment by oSumAtrIX** [comment]
> > I think you should align the buttons on the Right side
> > Like this:
> > IMG_20230607_093049
> 
> The position is purposefully not aligned the same so users don't spam through accept

I don't think it's right to motivate the alignment like that. Should be checked with M3

**Comment by 404-Program-not-found** [comment]
fair enough, will update in a sec

File: src/routes/+layout.svelte [hunk 1]
```diff
@@ -71,25 +90,61 @@
...
 </svelte:head>
 
 <QueryClientProvider client={queryClient}>
 	<NavHost />
+	<!-- if cookie consent hasn't been set -->
+	<Dialogue bind:modalOpen={askCookieConsent} dismissible={false}>
+		<svelte:fragment slot="title">It's your choice</svelte:fragment>
+		<svelte:fragment slot="description">
+			We use analytics and cookies to help us better understand how you use the site. Opting in is
+			optional and won't impact your experience.
```
Comment (oSumAtrIX) [inline src/routes/+layout.svelte:141]:
```suggestion
			This site uses analytics to understand better how you use it. Opting in is
			optional and won't impact your experience.
```

**Comment by alexandreteles** [comment]
What is the reason for it to overlay the whole page? Cookie consent is purposefully put in the footer and without blocking navigation so it doesn't break neither UI or UX. It should also not block interaction with other elements so it doesn't break if adblockers hide it.

**Comment by alexandreteles** [comment]
@xafn Did you remember to add the privacy and cookie pages before merging?

**Pull request merged**